app/server/lib/GranularAccess.ts

```diff
@@ -2376,6 +2376,14 @@ export class GranularAccess implements GranularAccessForBundle {
     this._pruneColumns(action, permInfo, tableId, accessCheck);
   }
 
+  private async _checkAttachmentAccess(docSession: OptDocSession, attId: number) {
```

Comment:
Can you talk me through what this fixes?

Reply:
Actually the lazy `&&` does the same thing :grimacing: